pg-ragtime/src/pg/ragtime.clj

```diff
@@ -0,0 +1,29 @@
+(ns pg.ragtime
+  (:require
+   [ragtime.core :as ragtime]
+   [ragtime.strategy]
+   [ragtime.protocols]
+   [pg.core :as pg])
+  (:import org.pg.Connection))
+
+(defn ensure-migrations-table!
+  [conn]
+  (pg/query conn "
+create table if not exists ragtime_migrations(
```

Comment:
Here, the name of the table is hardcoded. We can keep it as is for now but later on, it should be a configuration parameter.